src/tools.py

```diff
@@ -173,7 +182,8 @@ def b58_decode(s):
     for c in s:
         n *= 58
         if c not in b58_digits:
-            raise InvalidBase58Error('Character %r is not a valid base58 character' % c)
+            raise InvalidBase58Error('Character %r is not a ' +
```

Comment:
I must admit I'm not a big fan of the 80 characters per line rule. It made sense in the 90s, but now who has an 80 characters dispay? When a guide/rule/law is obsolete or produces bad result, it is meant to be ignored. :p Anyway, the 2nd folding is ok for me.